examples/module/mpc/train_mpc.py

```diff
@@ -0,0 +1,125 @@
+import os
+import mpc
+import time
+import torch
+import shutil
+import argparse
+import setproctitle
+import pypose as pp
+import pickle as pkl
+import torch.optim as optim
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--n_batch', type=int, default=16)
+    parser.add_argument('--n_state', type=int, default=5)
+    parser.add_argument('--n_ctrl', type=int, default=5)
+    parser.add_argument('--T', type=int, default=5)
+    parser.add_argument('--save', type=str)
+    parser.add_argument('--work', type=str, default='work')
+    parser.add_argument('--no-cuda', action='store_true')
```

Comment:
right now, I can only get the --no-cuda option work....the gpu option always complaining about some variables are on different devices., e.g. L136 in lqr.py